client/app/components/packages/working-package/edit.hbs

```diff
@@ -0,0 +1,109 @@
+<SaveableForm
+  @model={{@package}}
+  @validators={{array (hash) this.validations.SubmittablePackageFormValidations}}
```

Comment:
Oh good catch... I need to retrofit EAS, EIS, Tech memo with this validation.

Reply:
@godfreyyeung yeah takes a few steps  . thx for review.

Reply:
Sorry I meant to be keep forgetting to hit "approve", got sidetracked with another conversation. Not intentional, I promise 🙈